src/app/(admin)/admin/_components/Specialist/Contacts.js

```diff
@@ -2,20 +2,40 @@ import React from 'react';
 import { FormFieldWrapper } from '@admin/components/FormFieldWrapper';
 import { SpecialistFormFields, SpecialistFormSections } from '@admin/_lib/specialistData';
 import { TextInputList } from '@admin/components/TextInputList';
+import { useWatch } from 'react-hook-form';
+import { DisplayMatchingEntity } from '@admin/components/DisplayMatchingEntity';
+import { useFindMatchingEntities } from '@admin/_hooks/common';
 
 export function Contacts() {
-  const { phone, email, website, instagram, facebook, youtube, linkedin, tiktok, viber, telegram } =
-    SpecialistFormFields;
+  const PHONE = 'phone';
+  const EMAIL = 'email';
```

Comment:
Move these constants somewhere else to share between components, I see you duplicated it twice. But if you fix comment below, it won't be required as you will have only 1 component

Reply:
Put those ones into consts.js as I'm using both of them in multiple files